### PR TITLE
Adding fix for console.log on prod and some cleanups

### PR DESCRIPTION
**Pull request by rodriguezmanu**
Fixes https://github.com/moduslabs/modir/issues/88

### CODE REVIEW

**Review by Mrkezii — APPROVED** [review]

**Comment by grgur** [comment]
instead of `#88` you could have posted `closes #88` for a little bit of extra automation 😉

File: src/index.tsx [hunk 1]
```diff
@@ -63,6 +63,12 @@ if ((!navigator.onLine || process.env.NODE_ENV === 'development') && localStorag
   initApp()
 }
 
+// disable console log for prod
+if (process.env.NODE_ENV !== 'development') {
+  /* eslint-disable */
+  console.log = () => {}
```
Comment (grgur) [inline src/index.tsx:69]:
```suggestion
  const consoleMock = () => {}
  console.log = consoleMock;
  console.error = consoleMock;
  console.warn = consoleMock;
```

How about this so most common console methods are covered?

File: src/service/Data.tsx [hunk 2]
```diff
@@ -281,9 +281,6 @@ export const getModiteData = (slackId: string): GetModiteData => {
           // eslint-disable-next-line no-console
           console.log('error', e)
```
Comment (grgur) [inline src/service/Data.tsx:282]:
```suggestion
          console.error('Error fetching Modite data', e)
```

File: src/index.tsx (hunk 1 above)
Comment (rodriguezmanu) [reply src/index.tsx]: @grgur Updated!

File: src/service/Data.tsx (hunk 2 above)
Comment (rodriguezmanu) [reply src/service/Data.tsx]: @grgur Updated!

File: src/index.tsx (hunk 1 above)
Comment (rodriguezmanu) [reply src/index.tsx]: for some reason that i am investigating these changes are not good for ci circle gimbal https://app.circleci.com/pipelines/github/moduslabs/modir/478/workflows/0210b794-dbd7-4711-a65f-52c2a410d6bf/jobs/1669

Comment (rodriguezmanu) [reply src/index.tsx]: warn is not quite good for gimbal, I am not so familiar with this internal process and logs are not so helpful , any recommendation ?

**Pull request merged**